scipy/_build_utils/system_info.py

```diff
@@ -30,8 +30,8 @@ def combine_dict(*dicts, **kw):
                 elif value == old_value:
                     continue
 
-                raise ValueError("Conflicting configuration dicts: {!r} {!r}"
-                                 "".format(new_dict, d))
+                raise ValueError(f"Conflicting configuration dicts: {new_dict!r} {d!r}"
+                                 "")
```

Comment:
This was a bit over the ugliness tolerance threshold.
```suggestion
                message = f"Conflicting configuration dicts: {new_dict!r} {d!r}"
                raise ValueError(message)
```